packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -33,8 +33,6 @@ export const {
   retryLaneExpirationMs,
   syncLaneExpirationMs,
   transitionLaneExpirationMs,
-  enableFastAddPropertiesInDiffing,
-  enableViewTransition,
```

Comment:
Keep `enableViewTransition`, probably a rebase merge error since I had it briefly removed.